src/web/components/Onwards/Carousel/Carousel.tsx

```diff
@@ -607,9 +621,14 @@ export const Carousel: React.FC<OnwardsType> = ({
 							format,
 							palette: trailPalette,
 							webPublicationDate,
-							image: imageUrl,
+							image: fallbackImageUrl,
+							carouselImages,
 							kickerText,
 						} = trail;
+						const imageUrl =
+							isFullCardImage && carouselImages
+								? carouselImages['460']
+								: fallbackImageUrl;
```

Comment:
👍 this makes sense. It's basically correct in that this represents exactly what we're doing